src/c_kzg_4844.h

```diff
@@ -121,6 +121,8 @@ C_KZG_RET verify_kzg_proof(bool *out,
     const KZGProof *kzg_proof,
     const KZGSettings *s);
 
+void hash(uint8_t md[32], const uint8_t input[], size_t n);
```

Comment:
why does this need to be exposed?

Reply:
OK I saw your comment. I prefer to not include it in the interface if you're only using it for tests. If you think it's very valuable to have it, then I would want more comments in the header file and readme saying that it's only there for tests. But preferably just remove it.